api/build.gradle.kts

```diff
@@ -1,24 +1,35 @@
 plugins {
-    id("org.springframework.boot") version "2.2.6.RELEASE"
+    id("org.springframework.boot") version "2.3.0.RC1"
```

Comment:
Why do you want to use RC instead of the stable version?

Reply:
Spring Data r2dbc _Query Derivation_ is only available in the next release in mid of May. At this moment we can use RC version I suppose. And in a few weeks, we will move on RELEASE version